src/Microsoft.AspNetCore.Server.IISIntegration/IISDefaults.cs

```diff
@@ -5,6 +5,7 @@ namespace Microsoft.AspNetCore.Server.IISIntegration
 {
     public class IISDefaults
     {
+        public static readonly string AuthenticationScheme = "Windows";
```

Comment:
Are we going to change this in the future?  Just asking why this is diff than Negotiate/Ntlm